ledger/participant-integration-api/src/main/scala/platform/apiserver/services/LedgerConfigProvider.scala

```diff
@@ -164,11 +164,9 @@ private[apiserver] final class LedgerConfigProvider private (
         .map {
           case state.SubmissionResult.Acknowledged =>
             logger.info(s"Initial configuration submission $submissionId was successful")
-          case state.SubmissionResult.NotSupported =>
-            logger.info("Setting an initial ledger configuration is not supported")
```

Comment:
@mziolekda pointed out that removing this might cause issues with ledgers + drivers such as Canton that don't support submitting configuration. As these drivers have other ways of managing configuration, I'm going to change `LedgerConfiguration` to allow them to state that we should _never_ submit a new configuration, instead just waiting indefinitely for one to show up. @oliverse-da has agreed this approach will work for Canton.

I'll make this change separately.